src/pages/ReimbursementAccount/ACHContractStep.js

```diff
@@ -114,45 +153,38 @@ class ACHContractStep extends React.Component {
             || (_.size(this.state.beneficialOwners) === 3 && !this.state.ownsMoreThan25Percent);
     }
 
-    /**
-     * Clear the error associated to inputKey if found and store the inputKey new value in the state.
-     *
-     * @param {Integer} ownerIndex
-     * @param {Object} values
-     */
-    clearErrorAndSetBeneficialOwnerValues(ownerIndex, values) {
-        this.setState((prevState) => {
-            const beneficialOwners = [...prevState.beneficialOwners];
-            beneficialOwners[ownerIndex] = {...beneficialOwners[ownerIndex], ...values};
-            BankAccounts.updateReimbursementAccountDraft({beneficialOwners});
-            return {beneficialOwners};
-        });
-
-        // Prepare inputKeys for clearing errors
-        const inputKeys = _.keys(values);
-
-        // dob field has multiple validations/errors, we are handling it temporarily like this.
-        if (_.contains(inputKeys, 'dob')) {
-            inputKeys.push('dobAge');
+    submit(values) {
+        if (!this.validate(values)) {
+            return;
```

Comment:
We can remove this check. The Form component already does that [here](https://github.com/Expensify/App/blob/b816f5bd66384ad65633f7d74f6bf05a78bf79c5/src/components/Form.js#L138-L141) in the submit method.